x-pack/plugins/cases/common/constants.ts

```diff
@@ -103,14 +103,17 @@ export const GENERAL_CASES_OWNER = APP_ID;
 
 export const OWNER_INFO = {
   [SECURITY_SOLUTION_OWNER]: {
+    appId: 'securitySolutionUI',
```

Comment:
Yeah that's the only quick solution I could think of too. I suppose in the future we could create a registration when apps want to embed the cases page and they'd register their owner and app id?